### PR TITLE
Replace validation banner with GOV.UK Frontend error summary component

### PR DESCRIPTION
**Pull request by agalamatis**
trello: https://trello.com/c/qUAR9Nm4/20-1-replace-validation-banner-with-govuk-frontend-error-summary-component-in-the-briefs-frontend

### CODE REVIEW

File: tests/main/views/test_outcome.py [hunk 1]
```diff
@@ -150,10 +150,11 @@ def test_award_brief_post_raises_400_if_required_fields_not_filled(self):
         assert res.status_code == 400
 
         validation_message = document.xpath('//span[@class="validation-message"]')[0].text_content()
-        assert validation_message.strip() == "You need to answer this question."
+        assert validation_message.strip() == "Select a supplier."
 
         error_span = document.xpath('//span[@id="error-brief_response"]')[0]
-        assert self._strip_whitespace(error_span.text_content()) == "Youneedtoanswerthisquestion."
+        # assert that a framework iteration specific message has been rendered
+        assert len(self._strip_whitespace(error_span.text_content())) > 0
```
Comment (katstevens) [inline tests/main/views/test_outcome.py:157]: If we don't care about the content of the message, do we need the `strip_whitespace`? Can we just assert that exactly 1 error message has been shown?

**Comment by lfdebrux** [comment]
I'm just looking at these now, sorry I haven't been able to beforehand.

One small comment straight off the bat, the commit messages could be more descriptive, especially the first one. Something like the PR title would be good!

File: app/templates/create_buyer/create_buyer_account.html [hunk 2]
```diff
@@ -20,9 +20,13 @@
...
-{% with lede = "There was a problem with the details you gave for:" %}
-  {% include "toolkit/forms/validation.html" %}
-{% endwith %}
+<h1 class="govuk-heading-l">
+  Create a buyer account
+</h1>
+
+<p class="govuk-body-l">
+  You must be employed by, or represent, a public sector organisation.
+</p>
```
Comment (lfdebrux) [inline app/templates/create_buyer/create_buyer_account.html:29]: This content was moved in a previous PR but appears to have come back in a merge conflict

```suggestion
```

**Comment by lfdebrux** [comment]
I'm getting an issue where the error summary doesn't include the error messages; maybe we're missing a `govuk_errors` call somewhere?

image

**Comment by lfdebrux** [comment]
Looks like the other views are good though 👍

**Comment by lfdebrux** [comment]
Thanks for fixing those issues, they look good to me now.

Unfortunately I just found another one that was missed, at `app/main/views/supplier_questions.py:87`

File: app/main/forms/cancel.py [hunk 3]
```diff
@@ -10,7 +10,7 @@ class CancelBriefForm(FlaskForm):
         ('unsuccessful', 'There were no suitable suppliers'),
     ]
     cancel_reason = RadioField(
-        validators=[validators.InputRequired(message='You need to answer this question.')],
+        validators=[validators.InputRequired(message='Select a reason.')],
```
[the remark this answers is not in the document]
Comment (lfdebrux) [reply app/main/forms/cancel.py:13]: @agalamatis don't forget to ask @NoraGDS for advice about this!

[the remark this answers is not in the document]
Comment (NoraGDS) [reply app/main/forms/cancel.py:13]: How about: Select a reason for cancelling the brief

[the remark this answers is not in the document]
Comment (agalamatis) [reply app/main/forms/cancel.py:13]: Changed, rebasing, squashing and pushing now.

**Pull request merged**